sdk/textanalytics/Azure.AI.TextAnalytics/src/TextAnalyticsAudience.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.AI.TextAnalytics
+{
+    /// <summary> Cloud audiences available for ACR. </summary>
+    public readonly partial struct TextAnalyticsAudience : IEquatable<TextAnalyticsAudience>
+    {
+        private readonly string _value;
+
+        /// <summary> Determines if two <see cref="TextAnalyticsAudience"/> values are the same. </summary>
+        /// <exception cref="ArgumentNullException"> <paramref name="value"/> is null. </exception>
+        public TextAnalyticsAudience(string value)
+        {
+            _value = value ?? throw new ArgumentNullException(nameof(value));
+        }
+
+        private const string AzureResourceManagerChinaValue = "https://cognitiveservices.azure.cn/.default";
+        private const string AzureResourceManagerGovernmentValue = "https://cognitiveservices.azure.us/.default";
+        private const string AzureResourceManagerPublicCloudValue = "https://cognitiveservices.azure.com/.default";
+
+        /// <summary> Azure China. </summary>
+        public static TextAnalyticsAudience AzureResourceManagerChina { get; } = new TextAnalyticsAudience(AzureResourceManagerChinaValue);
+
+        /// <summary> Azure Government. </summary>
+        public static TextAnalyticsAudience AzureResourceManagerGovernment { get; } = new TextAnalyticsAudience(AzureResourceManagerGovernmentValue);
```

Comment:
should this be USGovernment?

Reply:
I'm following the pattern shown here https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/containerregistry/Azure.Containers.ContainerRegistry/src/Models/ContainerRegistryAudience.cs#L23

Reply:
Personally, I'm not a fan of the names used by ACR.   I'd prefer that we align them with resources likely to be more meaningful to customers.  Something like:

- AzureChina 
- AzureUsGovernment
- Azure

When you look through the Azure documentation, the landing pages don't refer to "Resource Manager" at all, they say things like:

-  [Azure for US Government](https://azure.microsoft.com/en-us/global-infrastructure/government/)
- [Microsoft Azure operated by 21Vianet](https://docs.microsoft.com/en-us/azure/china/overview-operations) _(this is the China cloud landing page)_
- [Azure China developer guide](https://docs.microsoft.com/en-us/azure/china/resources-developer-guide)
- [What is Azure?](https://azure.microsoft.com/en-us/overview/what-is-azure/)  _(public cloud landing page)_

Reply:
We've also got https://apiview.dev/Assemblies/Review/6a40db7f9eeb41dabfe5768f1e796c81#Azure.Identity.AzureAuthorityHosts for prior art that I think started these naming patterns.

Reply:
I've no objections to aligning with the Identity hosts values; I find the naming there to address my concerns.

Reply:
Also, these names shouldn't contain ResourceManager. I know that the values in ACR do, but that was because they are specifically using the ARM audience and they are planning on adding ACR specific Audience values, so ResourceManager was added to them to distinguish.